README.md

```diff
@@ -67,7 +67,7 @@ module.exports = {
 
 This plugin provides one config:
 
-- `plugin:regexp/recommended` ... This is the recommended configuration for this plugin.  
```

Comment:
My editor automatically trims trailing spaces. Should I revert these changes?

Reply:
If we want to break a line, you need two spaces, but I think we don't have to break a line here 😄

Reply:
Is that a common Markdown feature or specific to GH-flavor Markdown? 

I always use `<br>` for line breaks btw. It's more explicit.